src/main/java/ch/njol/skript/ScriptLoader.java

```diff
@@ -1019,12 +1013,14 @@ public static ArrayList<TriggerItem> loadItems(SectionNode node) {
 					}
 					continue;
 				} finally {
+					RetainingLogHandler afterParse = handler.backup();
+					handler.clear();
 					handler.printLog();
+					if (item != null && (Skript.debug() || subNode.debug()))
+						Skript.debug(SkriptColor.replaceColorChar(parser.getIndentation() + item.toString(null, true)));
+					afterParse.printLog();
```

Comment:
```suggestion
					if (item != null && (Skript.debug() || subNode.debug()))
						Skript.debug(SkriptColor.replaceColorChar(parser.getIndentation() + item.toString(null, true)));
					handler.printLog();
```
I think we could just do this? If you can test